### ISSUE TITLE
circumflex (hat, bar, etc.) following unicode symbol displays visually as consecutive characters in Atom

### ISSUE THREAD

**Issue by John-Boik** [JunoLab/atom-latex-completions#43]
When I enter a unicode character in Atom, for example, /alpha, using either tab completions or Ctrl+U 03B1, and follow it by a circumflex, for example, /hat or Ctrl+U 0302, Atom displays it as one character but it visually looks like an alpha followed by a hat. That is, the hat is not visually placed over the alpha, but as a next character. This same thing occurs if I paste a correct /alpha /hat symbol into Atom from the Julia REPL or from another location. If I paste a copied /alpha /hat into Gedit, for example, it will display correctly.  If I copy the /alpha /hat from Atom editor and paste it into Gedit or REPL, it displays correctly.

I assume this is due to something in Atom, not necessarily atom-latex-completions, but perhaps others here have seen the same issue and have found a solution.  

System:
Ubuntu 19.10
Atom 1.41.0

John

**Comment by pfitzseb** [comment]
This is a font issue, I think. Try e.g. `Fira Mono`, which displays your example fine for me:
![grafik](https://user-images.githubusercontent.com/6735977/85988337-2ce8d480-b9ef-11ea-8804-2fe8c3c4993d.png)

**Comment by John-Boik** [comment]
Thanks @pfitzseb. There must be something else going on, as changing fonts helped but did not fix the problem on my machine. In Adam editor Settings page, I changed the Font Family to Fira Mono, as suggested. But as per this screenshot, 
![fonts](https://user-images.githubusercontent.com/22616258/86026831-2c4f3e80-b9ed-11ea-9123-ae588270ec37.png)
the circumflex is still not centered over the initial symbol, and is hard to distinguish if the initial symbol is /beta.  I also tried other fonts that are supposed to work well with a circumflex, like Andale Mono and Courier New, but they too have the same problem.

Any other suggestions?
John
